### ISSUE TITLE
Configuration options for nested destructuring

### ISSUE THREAD

**Issue by joncursi** [lukeapage/eslint-plugin-destructuring#41]
Lots of info on this rule request here:

* https://github.com/eslint/eslint/issues/7973
* https://github.com/eslint/eslint/issues/7973#issuecomment-280716417

Quite a few people looking to see this in a plugin. Let me know what you think!

**Comment by lukeapage** [comment]
I agree that it most makes sense as an extension of the existing eslint rule - prefer destructuring. Otherwise it would be re-implementing alot of the same logic I expect. I created this plugin to cover a few cases where I wanted a rule but it was deemed too obscure for eslint core. I'm happy to accept any sensible contributions to this plugin though.
